Runtime/RuntimeUtil.cs

```diff
@@ -2,7 +2,11 @@
 using System.Collections.Generic;
 using JetBrains.Annotations;
 using UnityEngine;
+using UnityEngine.SceneManagement;
+
+#if VRC_SDK_VRCSDK3
```

Comment:
Is it okay to add a version define like `NDMF_VRCSDK3_AVATARS` for "com.vrchat.avatars" to asmdef? (naming follows lilToon)

Reply:
Yes, that would be fine I think.

Reply:
In fact, I think it's necessary - I don't think the defines auto-set by the VRCSDK are a stable API.